std/internal/math/gammafunction.d

```diff
@@ -231,7 +238,12 @@ real igammaTemmeLarge(real a, real x)
 
 public:
 /// The maximum value of x for which gamma(x) < real.infinity.
-enum real MAXGAMMA = 1755.5483429L;
+static if (real.mant_dig == 64)  // 80-bit reals
```

Comment:
Indeed I tried, but floatTraits is still private. I agree it'd be better to use floatTraits.
When is #3011 going to be merged?